### PR TITLE
Added rumbles

### LINKED ISSUE #60: Fix rumble on final
Final bot rumble is broken since the configureLEDs function is gone, use a trigger & callback

### PR DESCRIPTION
**Pull request by ThatXliner**
- Shooter when it reaches target velocity
- Intake when beam broken

- Closes #60

### CODE REVIEW

**Comment by realjoshuau** [comment]
I need to fix the /format command.....

File: src/main/java/frc/robot/subsystems/shooter/Shooter.java [hunk 1]
```diff
@@ -74,9 +76,15 @@ public Command setVoltage(double voltage, double followerVoltage) {
         .finallyDo(shooterIO::off);
   }
 
+  // Only cares about main motor
+  public boolean isAtTargetVelocity() {
+    return Math.abs(getVelocity() - targetVelocity) < ShooterConstants.kVelocityTolerance;
```
Comment (chaoticthegreat) [inline src/main/java/frc/robot/subsystems/shooter/Shooter.java:81]: I like epsilon equals

File: src/main/java/frc/robot/RobotContainer.java [hunk 2]
```diff
@@ -323,6 +323,16 @@ private void configureRumble() {
                                     }));
 
         }
+        if (FeatureFlags.kIntakeEnabled) {
+            // XXX: Or maybe I should re-use the debouncedBeamBreak trigger inside this
+            // object
+            new Trigger(() -> intake.isBeamBroken())
+                    .onTrue(new InstantCommand(() -> {
```
Comment (chaoticthegreat) [inline src/main/java/frc/robot/RobotContainer.java:330]: I think Commands.runOnce looks cleaner

File: src/main/java/frc/robot/subsystems/shooter/Shooter.java (hunk 1 above)
Comment (ThatXliner) [reply src/main/java/frc/robot/subsystems/shooter/Shooter.java:81]: I agree. Will fix

File: src/main/java/frc/robot/RobotContainer.java (hunk 2 above)
Comment (ThatXliner) [reply src/main/java/frc/robot/RobotContainer.java:330]: Alr

**Review by realjoshuau — APPROVED** [review]

**Pull request merged**